### ISSUE TITLE
Rebalance queues after node restart

### ISSUE THREAD

**Issue by mkuratczyk** [rabbitmq/cluster-operator#168]
When a node is restarted, it will not become the leader for any queues unless new queues are created or another node leaves the cluster. That leads to suboptimal performance. We should run `rabbitmq-queues rebalance all` after a new node joins the cluster. Ideally, if we know all nodes will be restarted in quick succession, we can only run it once at the end only. We can discuss with the team to better understand how important it is avoid excessive rebalances (perhaps it's not a problem in practice and such optimization is not worth it). 

A thread about when rebalancing should be executed:
https://vmware.slack.com/archives/C055BSG8E/p1592553301313100

TL;DR
* after a successful deployment
* after a node restart but ideally not too aggressively (multiple rebalances on a busy cluster is a bad idea)
* if executed without an explicit event (eg. periodically) then it should be opt-in behaviour

There is a similar story #90. This one requires a command to be executed only after RabbitMQ is upgraded which is harder to detect but perhaps the learnings from this issue can be applied there.
